src/core/surface/completion_queue.c

```diff
@@ -45,6 +45,13 @@
 #include <grpc/support/atm.h>
 #include <grpc/support/log.h>
 
+#define MAX_PLUCKERS 4
```

Comment:
So, I have a question. How is the MAX_PLUCKER policy actually enforced? Is it part of the cq API that there would never be more than this many pluckers? Could our code screw this up somehow?